.php_cs.dist

```diff
@@ -21,6 +21,11 @@ $finder = PhpCsFixer\Finder::create()
         $src.'/addons/tests',
         $src.'/addons/users',
     ])
+    ->append([
+        __FILE__,
```

Comment:
Das wird glaube ich nichts bringen, da der CS-Fixer sich auf Dateien `*.php` beschränkt.

Reply:
Lt. Prettyci zumindest wurde es bemäkelt

Reply:
Ah interessant. Dann haben [diese beiden Zeilen](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.15/src/Finder.php#L29-L30) scheinbar nur Auswirkung auf die Dateien in den eingebundenen Ordnern, und die hier explizit eingebundene Datei ist davon unabhängig.

Reply:
Übrigens von symfony geklaut